docs/Documentation/Environment/shell.md

```diff
@@ -280,7 +293,7 @@ if [ -f ~/MYENV/mymods ];    then . ~/MYENV/mymods ;    fi
 if [ -f ~/MYENV/myfuncs ];   then . ~/MYENV/myfuncs ;   fi
 ```
 
-to test things.  After they are working as desired then add this lines to your .bashrc file.  You can add a # to the lines in your .bashrc file to disable them.  
+to test things.  After they are working as desired then add these lines to your .bashrc file.  You can add a # to the lines in your .bashrc file to disable them.  
 
 There are copies of the default .bashrc and .bash_profile files in
```

Comment:
I think the "Some Commands" section could be formatted to be more readable, maybe a table similar to what we have on the lmod page?